src/Services/AddCommentService.php

```diff
@@ -1,29 +1,24 @@
 <?php
 
-namespace DrupalRector\Utility;
+namespace DrupalRector\Services;
 
 use PhpParser\Comment;
 use PhpParser\Node;
 use Rector\NodeTypeResolver\Node\AttributeKey;
-use Symplify\PackageBuilder\Parameter\ParameterProvider;
 
 /**
  * Provides an easy way to add a comment to a statement.
  */
-trait AddCommentTrait
+class AddCommentService
 {
 
     protected bool $noticesAsComments = false;
 
     /**
-     * @param array<string,bool>|array<class-string,object> $configuration
-     *
-     * @return void
+     * @param bool $noticesAsComments
      */
-    protected function configureNoticesAsComments(array &$configuration): void
-    {
-        $this->noticesAsComments = $configuration['drupal_rector_notices_as_comments'] ?? false;
-        unset($configuration['drupal_rector_notices_as_comments']);
+    public function __construct(bool $noticesAsComments = true) {
+        $this->noticesAsComments = $noticesAsComments;
     }
```

Comment:
We never configure it differently, should we just remove the argument?

Reply:
It needs disabling in update bot. At least that is how it always worked.